Formula/sipp.rb

```diff
@@ -22,4 +21,8 @@ def install
     system "make", "DESTDIR=#{prefix}"
     bin.install "sipp"
   end
+
+  test do
+    assert_equal "SIPp", shell_output("#{bin}/sipp -v | grep SIPp | awk '{ print $1 }'").strip
```

Comment:
Ahh, it's because sipp -v exits with a return code 99 instead of 0. Wasn't paying attention to the detail of the output.